docker_templates/templates/docker_images_ros2/nightly/create_ros_image.Dockerfile.em

```diff
@@ -67,10 +68,15 @@ ARG ROS2_BINARY_URL=@ros2_binary_url
 RUN wget -q $ROS2_BINARY_URL -O - | \
     tar -xj --strip-components=1 -C /opt/ros/$ROS_DISTRO
 
-# install setup files
-RUN apt-get update && apt-get install -q -y \
-    ros-$ROS_DISTRO-ros-workspace \
-    && rm -rf /var/lib/apt/lists/*
+# Overwrite setup scripts with ones that point to /opt/ros/$ROS_DISTRO
+RUN mkdir -p /tmp/rosws/build \
```

Comment:
I still wouldn't call this `rosws` since it is not a ROS workspace.

Reply:
Ah I didn't mean that. The folder name can be a random ( `rosws` was fewer characters than ros_workspace), but gives me something to `rm -r` at the end.

Reply:
Haha, got it. Yeah `ros_workspace` would be perfect.

Reply:
called it `dir` in 5dd6884